src/app/content/search/components/SearchResultsSidebar/__snapshots__/index.spec.tsx.snap

```diff
@@ -870,6 +870,7 @@ exports[`SearchResultsSidebar matches snapshot with related key terms 1`] = `
       </h3>
       <a
         aria-current={true}
+        ariaLabel="undefined in test-page-1"
```

Comment:
i think you have to put your ariaLabel in state for it to not show this on the first render, because the ref will be empty until it renders the first time

Reply:
Right. Have to do the set in a useEffect.